src/backend/renderer/multigpu/mod.rs

```diff
@@ -1212,6 +1212,7 @@ where
                             Rectangle::from_loc_and_size((0, 0), buffer_size).to_f64(),
                             Rectangle::from_loc_and_size((0, 0), self.size),
                             &damage,
+                            &[],
```

Comment:
Well I guess, everything is opaque in this case, but it doesn't matter because we cleared it before?

Reply:
Right, I added the size as opaque here. We might want to use a different shader in future for drawing opaque regions ignoring the alpha channel.

(I also changed it in the same function for the mem copy case)